releases/ingress-nginx/default.yaml

```diff
@@ -34,6 +34,7 @@ _set:
 
   service:
     annotations: {}
+    externalTrafficPolicy: Cluster
```

Comment:
Why don't we use most wanted `Local` here, f.e. to preserve customer IP?

Reply:
It was my suggestion, because the default is ClusterLocal.There are things like proxy-protocol, x-* tags which just don't require local... Btw the base chart also does not override the default. WDYT?

Reply:
I agree, we should not override chart default by default :)